simplepath/utils.py

```diff
@@ -0,0 +1,35 @@
+"""Module that describes all utility functions for Simplepath."""
+
+
+def convert_object_to_dict(data_object):
```

Comment:
@gregarmer suggested to call this `deepvars` which I like much better. please rename